vis/templates/includes/_local-search.jade

```diff
@@ -2,7 +2,7 @@
 
 .SearchForm.js-PCCSearch(class="{% if modifier %}SearchForm--{{ modifier }}{% endif %}")
   if modifier == 'hero'
-    img.SearchForm-img(src="{% static 'images/homepage-background.png' %}", srcset="{% static 'images/homepage-background@2x.png' %} 2x")
+    div.SearchForm-img(src="{% static 'images/homepage-background.png' %}", srcset="{% static 'images/homepage-background@2x.png' %} 2x")
```

Comment:
We don't need these attributes anymore as we were previously using an image tag within the HTML.

I wonder if we can move the CSS to the parent element with the modifier too (`.SearchForm--hero`).

We can then drop this whole block of code.

Reply:
Moved up as suggested in the latest fixup commit!